lib/charms/mongodb/v1/mongodb_provider.py

```diff
@@ -99,8 +103,12 @@ def pass_sanity_hook_checks(self) -> bool:
 
         return True
 
-    def pass_hook_checks(self, event: EventBase) -> bool:
+    def pass_hook_checks(self, event: RelationChangedEvent) -> bool:
```

Comment:
this should be RelationEvent or something of that sort, from what I can tell it also gets called on `relation_broken` events

Reply:
Oh my bad, it's an error in typing indeed.